content/diversity-accessibility.html

```diff
@@ -0,0 +1,92 @@
+
+<p>We will be making every effort to hold as inclusive an event as possible, and to shift the demographic of our audience and speakers. This is not just a vague aim, <a href="/news/target">it's a clear target</a>.</p>
+
+<h3>Code of conduct</h3>
+<p>Like every PyCon, we have a prominent <a href="/codeofconduct">code of conduct</a> <strong>that will be enforced</strong>.</p>
+
+<h3>Minorities in our community</h3>
+<p>It’s not simply iniquitous that the minorities of our community - for example, women, people of non-European descent, people with disabilities - find it harder to participate in it: their participation is vital to its health and needed now more than ever.</p>
+
+<p>If you belong to one of these or any other under-represented groups, we’re especially keen to have you at PyCon. We want to help redress the imbalance, make a clear statement about your value to the community affirm your position as a first-class citizen of it.</p>
+
+
+<h3>Django Girls</h3>
+<p>There'll be a day-long <a href="/djangogirls">Django Girls workshop</a> at PyCon. Since the first DjangoGirls event in July 2014, hundreds of women have been introduced to Django and Python through these free workshops.</p>
+
+<p>We are offering a number of free conference tickets for PyCon to DjangoGirls attendees. These are full tickets, and include meals, refreshments and everything else that the conference has to offer.</p>
+
+<p>Both the DjangoGirls workshop and the ticket offer will be oversubscribed; places and tickets will be allocated by the Django Girls organisers.</p>
+
+<h3>Trans*Code</h3>
+<p><a href="/transcode">Trans*Code</a> will be running a hackday at the event.</p>
+
+<p>The Trans*Code event is an opportunity to work with others on projects and issues that particularly affect the trans community.</p>
+
+<h3>Financial assistance</h3>
+<p>We will be supporting individuals who need help to meet the costs of travel, subsistence or even taking days off work. This will include discounted or free conference tickets. <em>Speakers will automatically be eligible for this assistance if they require it.</em></p>
+
+<h4>Apply for financial assistance</h4>
+<p>We have set aside a considerable sum for financial assistance.</p>
+
+<p><em>Don't be reluctant to apply for financial assistance.</em> Many of your fellow speakers and some members of the committee will be doing the same. The money is there to help <strong>you</strong>.</p>
+
+<h3>Supporter tickets</h3>
+<p>We have made <em>Diversity supporter tickets</em> available. As well as your own ticket, you can buy a ticket on behalf of an attendee who couldn’t otherwise afford it. These tickets will be allocated by an awards committee, and you or your organisation will be - unless you state otherwise - thanked publicly for your generous support.</p>
```

Comment:
Agree, we should.  There's no harm in asking!

However, could we call it something other than "Diversity supporter ticket"?  Maybe just "supporter ticket"?  I think the remit of the financial aid programme is broader than encouraging diversity.

@doismellburning will have views here -- he's looking at ticket pricing now.